src/generators.rs

```diff
@@ -437,10 +462,16 @@ pub fn directed_star_graph(
             .collect(),
     };
     for node in nodes[1..].iter() {
-        if inward {
+        //Add edges in both directions if bidirection is True
+        if bidirectional {
             graph.add_edge(*node, nodes[0], py.None());
-        } else {
             graph.add_edge(nodes[0], *node, py.None());
+        } else {
+            if inward {
+                graph.add_edge(*node, nodes[0], py.None());
+            } else {
+                graph.add_edge(nodes[0], *node, py.None());
+            }
```

Comment:
I think that this if can be collapsed:

```suggestion
        } else if inward {
                graph.add_edge(*node, nodes[0], py.None());
        } else {
                graph.add_edge(nodes[0], *node, py.None());
        }
```